CI/setup-windows-sdk.sh

```diff
@@ -122,6 +122,7 @@ while true; do
     "mingw-w64-${BUILDCOMPONENT}-lua-luarocks" \
     "mingw-w64-${BUILDCOMPONENT}-meson" \
     "mingw-w64-${BUILDCOMPONENT}-ninja" \
+    "mingw-w64-${BUILDCOMPONENT}-uasm" \
```

Comment:
Change the line above here - `cmake` depends on `ninja` (rather than the other way around) and your primary issue right now is that `cmake` is not present...
```diff
-    "mingw-w64-${BUILDCOMPONENT}-ninja" \
+    "mingw-w64-${BUILDCOMPONENT}-cmake" \
```